inc/classes/class_db_mysql.php

```diff
@@ -2,48 +2,30 @@
 
 class db
 {
-    public $mysqli = 0;
     public $link_id = 0;
     public $query_id = 0;
-    public $record   = array();
-    public $print_sql_error;
+    public $record = array();
     public $success = false;
     public $count_query = 0;
     public $errors = '';
     public $errorsFound = 0;
     public $connectfailure = 0;  //0= no error, 1=connection error, 2=database error
     public $QueryArgs = array();
 
-  // Construktor
-    public function __construct()
```

Comment:
Removing the constructor also removes the theck that the module for mysql(i) is loaded.
Not a huge issue by itself, it will just fail elsewhere and we need to make sure that the error is handled properly